site/content/blog/subpath.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Subpath Support"
+date: 2018-06-25T15:35:09-04:00
+draft: true
+---
+Mattermost 5.1 will include support for [serving Mattermost from subpaths](https://mattermost.atlassian.net/browse/MM-10366). This allows Mattermost to be exposed at something like https://example.com/company/mattermost, with your proxy server exposing different services at other subpaths. Subpath support is configured via the [Site URL](https://docs.mattermost.com/administration/config-settings.html?highlight=siteurl#site-url). 
+
+#### Using subpaths in production
+
+In production, after configuration and restart, the application server will expect all HTTP requests to be anchored to the configured subpath. This includes static assets, API calls, and websockets. For convenience, the application server will redirect any unexpected requests back into this subpath, e.g. forwarding http://example.com/login to http://example.com/company/mattermost/login.
+
+The pre-built static assets shipped with Mattermost assume being hosted at `/static/`. In production, when the `SiteURL` is configured with a subpath such as https://example.com/company/mattermost, the application server will rewrite any affected assets on startup to point at `/company/mattermost/static` instead. This rewriting can be done manually with the `mattermost config subpath` command:
+
+```
+mattermost config subpath --path /company/mattermost
+```
+
+Rewriting the static assets manually may be necessary when deploying to a CDN.
+
+#### Using subpaths in development
+
+In development, with webpack running, the static assets are rewritten whenever a source file changes. While the server is still configured via the `SiteURL`, webpack requires this to be set via an environment variable:
+```
+cd mattermost-webapp/
+make stop
+export MM_SERVICESETTINGS_SITEURL='http://localhost:8065/company/mattermost'
+make run
+```
+
+To resume testing at the root, either define a `SiteURL` with no subpath or unset the environment variable:
+```
+unset MM_SERVICESETTINGS_SITEURL
+make restart
+```
+
+Note that this environment variable is also honoured by the server as an override to config.json as described in [Configuration Settings](https://docs.mattermost.com/administration/config-settings.html#configuration-settings), so feel free to throw this export in your shell's initialization script to apply to both the server and the webapp simultaneously.
```

Comment:
Propose 

```
This environment variable is also honoured by the server as an override to config.json as described in [documentation](https://docs.mattermost.com/administration/config-settings.html#configuration-settings). Therefore, you can include this export in your shell's initialization script to apply to both the server and the webapp simultaneously.
```

^wondering if `shell's initialization script` is clear to non-English devs? 0/5

Reply:
Proposal sounds good. I think `shell` and `initialization script` are standard dev terms for `.bashrc` and `.zshrc` and the like, which would probably be familiar to non-English devs. I'm clearly using the force here :)